{
  "comments": [
    {
      "key": {
        "uuid": "a766b734_2a0bce30",
        "filename": "org.eclipse.jgit.test/tst/org/eclipse/jgit/api/CloneCommandTest.java",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 6
      },
      "writtenOn": "2014-12-11T13:13:14Z",
      "side": 1,
      "message": "You could also test the error situations like in the InitCommandTest",
      "revId": "0777fb41057c41a88ee0914bf60e186ac69a01d3",
      "serverId": "97ee7c02-f12f-4043-b43e-dea463d88b31",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "a766b734_8a7b0298",
        "filename": "org.eclipse.jgit/src/org/eclipse/jgit/api/CloneCommand.java",
        "patchSetId": 1
      },
      "lineNbr": 361,
      "author": {
        "id": 6
      },
      "writtenOn": "2014-12-11T13:13:14Z",
      "side": 1,
      "message": "This is not implemented. The exception is only thrown by the call method. Either also implement the check here or document the Exception in the call method.",
      "range": {
        "startLine": 358,
        "startChar": 4,
        "endLine": 361,
        "endChar": 26
      },
      "revId": "0777fb41057c41a88ee0914bf60e186ac69a01d3",
      "serverId": "97ee7c02-f12f-4043-b43e-dea463d88b31",
      "unresolved": false
    }
  ]
}